specification/ai/Azure.AI.Projects/agents/runs/models.tsp

```diff
@@ -276,6 +276,10 @@ model CreateRunOptions {
   @encodedName("application/json", "response_format")
```

Comment:
Please align with
https://github.com/Azure/azure-rest-api-specs/pull/30909/files

We shall update also ThreadRun model like below

@encodedName("application/json", "parallel_tool_calls")
  parallelToolCalls: boolean;

Reply:
Also difference in optional vs non-optional

Reply:
As discussed lets make parallelToolCalls non-optional aligned with OpenAI Assistant TypeSpec ThreadRun model

Reply:
Fixed.